xarray/computation/fit.py

```diff
@@ -474,7 +474,7 @@ def _wrapper(Y, *args, **kwargs):
             mask = np.all([np.any(~np.isnan(x), axis=0), ~np.isnan(y)], axis=0)
             x = x[:, mask]
             y = y[mask]
-            if not len(y):
+            if not y:
```

Comment:
`y` is an array, so perhaps this is better
```suggestion
            if y.size == 0:
```